packages/netlify/README.md

```diff
@@ -238,7 +241,7 @@ Netlify Functions requires binary data in the `body` to be base64 encoded. The `
 
 We check for common mime types for audio, image, and video files. To include specific mime types that should be treated as binary data, include the `binaryMediaTypes` option with a list of binary mime types.
 
-```js {12}
+```js
 // src/pages/image.jpg.ts
 
 import fs from 'node:fs';
```

Comment:
Maybe we can remove this extra line?

```suggestion
```js
// src/pages/image.jpg.ts
import fs from 'node:fs';
```